lib/cli/parse.js

```diff
@@ -9,13 +7,13 @@ const parser = require('yargs')('run --help')
       alias: 'globs',
     });
   })
+  .env('FORMAT')
```

Comment:
All for using environment variables for options.

Reply:
Since this environment variable lives in a global scope, can we choose a more specific name such as `PACKAGE_FORMAT_CONFIG`?

Reply:
@camacho Done.